source/annotations/provider_specific.go

```diff
@@ -0,0 +1,76 @@
+/*
+Copyright 2025 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package annotations
+
+import (
+	"fmt"
+	"strings"
+
+	"sigs.k8s.io/external-dns/endpoint"
+)
+
+func ProviderSpecificAnnotations(annotations map[string]string) (endpoint.ProviderSpecific, string) {
+	providerSpecificAnnotations := endpoint.ProviderSpecific{}
+
+	if v, exists := annotations[CloudflareProxiedKey]; exists {
```

Comment:
Shouldn't we do the same for cloudflare as we do for others like the prefix scan?
`external-dns.alpha.kubernetes.io/cloudflare-*`

Reply:
done